components/MapWrapper/StyledMapWrapper.js

```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components";
+
+export default styled.div`
+  position: relative;
+  height: 100vh;
+  width: 100vw;
+`;
```

Comment:
Are you sure you will need this wrapper?
Couldn´t you just assign 100vh/vw to your `Map/StyledMap.js` component?